### PR TITLE
Added new assets and fixed UI

### PR DESCRIPTION
**Pull request by neel-ds**
- Optimized UI/UX
- Added new assets
- Updated content

### CODE REVIEW

File: src/pages/index.jsx [hunk 1]
```diff
@@ -47,79 +45,85 @@ export default function Home() {
...
+                desc="lorem ipsium"
+              />
+              <Card
+                img="/syncX.png"
+                name="Sameul Bahir"
+                title="Content Creator"
+                desc="lorem ipsium"
+              />
+              <Card
+                img="/syncX.png"
```
Comment (senali-d) [inline src/pages/index.jsx:91]: @neel-ds Instead repeat component again and again. It is better we can loop through a json array.

**Review by senali-d — CHANGES_REQUESTED** [review]
Recheck and fix the responsive issues as well.

File: src/components/navbar.jsx [hunk 2]
```diff
@@ -1,27 +1,74 @@
...
     <>
-      <nav className="fixed z-10 w-full mx-auto px-2 sm:px-4 py-2.5 rounded drop-shadow-md bg-[#f1f1f1]">
+      <nav className={`fixed z-10 w-full mx-auto px-2 sm:px-4 py-3 rounded drop-shadow-md ${navbarBg} bg-opacity-90`}>
         <div className="max-w-[1080px] container flex flex-wrap justify-between items-center mx-auto">
           <Link href="/" className="flex items-center flex-1">
             <span className="flex flex-row items-center self-center text-2xl font-semibold whitespace-nowrap">
-              <Image src="/images/promate.svg" width="40" height="40" alt="Promate" className="mr-3" />
-              Promate
+              <Image
+                src="/syncX.png"
```
Comment (senali-d) [inline src/components/navbar.jsx:61]:
```suggestion
                src="/syncx.png"
```

File: src/pages/index.jsx [hunk 3]
```diff
@@ -32,94 +29,123 @@ const Card = ({ img, name, title, desc, style }) => {
   );
 };
 
+const cardData = [
+  {
+    id: 1,
+    img: "/syncX.png",
```
Comment (senali-d) [inline src/pages/index.jsx:35]:
```suggestion
    img: "/syncx.png",
```

**Comment by neel-ds** [comment]
- Fixed mobile view of NavBar
- Added new featured assets 
- Updated Home & About page

**Pull request merged**